apps/backend/src/room/logic/create-room.ts

```diff
@@ -0,0 +1,30 @@
+import { Event, CreateRoomCommand, CreateRoomEvent } from "@packages/domain";
+import { ResultAsync, errAsync } from "neverthrow";
+import { EventSource } from "~/models/event";
+import { error } from "~/utils";
+
+const EventSourceWriteError = error("EventSourceWriteError");
+export type EventSourceWriteError = ReturnType<typeof EventSourceWriteError>;
+
+type CreateRoomError = EventSourceWriteError;
+
+export interface Pass {
```

Comment:
@JohnsonMao 
Could you check the name of the type?

Reply:
Oops! Thank you for catching my mistake.